Header header
detector_2d/CoordBoxes[] detected_objects
